os/drivers/audio/alc5658.c

```diff
@@ -584,15 +588,13 @@ static int alc5658_configure(FAR struct audio_lowerhalf_s *dev, FAR const struct
 			/* Set the volume */
 
 			uint16_t volume = caps->ac_controls.hw[0];
-			audvdbg("    Volume: %d\n", volume);
-
-			if (volume >= 0 && volume <= 1000) {
-				/* Scale the volume setting to the range {0.. 63} */
 
-				alc5658_setvolume(priv, (63 * volume / 1000), priv->mute);
-			} else {
+			if (volume >= 0 && volume <= 31) {
```

Comment:
If 0 means min and 31 is max, then why don't you use #define??

Reply:
Or you'd better add 2 variable inside of priv to maintain min & max

Reply:
This variable just stores the value set through configuration. So just one is enough.
But as you said, the min & max values can be changed to MACROs. Will do.